src/test/java/suite/E2EPipelinePrivateOperatorTestSuite.java

```diff
@@ -0,0 +1,16 @@
+package suite;
+
+import org.junit.platform.suite.api.SelectClasses;
+import org.junit.platform.suite.api.Suite;
+import suite.basic.BasicTest;
+import suite.operator.V2ApiOperatorLocalOnlyTest;
+import suite.operator.V2ApiOperatorTest;
+
+@Suite
+@SelectClasses({
```

Comment:
Why do we have separate pipeline and non pipeline suites? Why does pipeline have a local only suite? I dislike needing two suites that do most of the same thing, can we collapse them into one?